packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -38,6 +42,12 @@ const IDP_CONFIG = {
  */
 const SocialLogin = ({idps}) => {
     const {formatMessage} = useIntl()
+    const authorizeIDP = useAuthHelper(AuthHelpers.AuthorizeIDP)
+
+    // Build redirectURI from config values
+    const appOrigin = useAppOrigin()
+    const redirectPath = getConfig().app.login.social?.redirectURI || ''
```

Comment:
NIT: It would be safer to have the optional chaining throughout this chain. E.g. `getConfig()?.app?.login?.social?.redirect`